File: SecondYear/jeezzzz_AjeeshRawal_2226cseai1021_2/Week_1/pattern9.java
class Solution {

    void printTriangle(int n) {
        int i,j,k,space=0;
        for(i=n;i>=1;i--)
        {
            for(j=1;j<=i;j++)
            {
                System.out.print("*");
            }
            for(k=1;k<=space;k++)
            {
                System.out.print(" ");
            }
            for(k=1;k<=space;k++)
            {
                System.out.print(" ");
            }
            space++;
            for(j=1;j<=i;j++)
            {
                System.out.print("*");
            }
            System.out.println();
        }
        space=n-1;
        for(i=1;i>=n;i++)
        {
            for(j=1;j<=i;j++)
            {
                System.out.print("*");
            }
            for(k=1;k<=space;k++)
            {
                System.out.print(" ");
            }
            for(k=1;k<=space;k++)
            {
                System.out.print(" ");
            }
            space--;
            for(j=1;j<=i;j++)
            {
                System.out.print("*");
            }
            System.out.println();
        }
        
    }
}

File: SecondYear/swarnkarvikash_VikashKumarSwarnkar_2226csit1034_2/Week_1/Pattern9.cpp
class Solution {
  public:
    void printTriangle(int n) {
        // code here
         int space=0;
        for(int i=0;i<n;i++)
        {
            for(int j=1;j<=n-i;j++)
            {
                cout<<"*";
            }
            for(int j=0;j<space;j++)
            {
                cout<<" ";
            }
             for(int j=1;j<=n-i;j++)
            {
                cout<<"*";
            }
            space+=2;
            cout<<endl;
        }
        
        space=2*n-2;
        
        for(int i=1;i<=n;i++)
        {
            for(int j=1;j<=i;j++)
            {
                cout<<"*";
            }
            for(int j=0;j<space;j++)
            {
                cout<<" ";
            }
             for(int j=1;j<=i;j++)
            {
                cout<<"*";
            }
            space-=2;
            cout<<endl;
        }
       
            
            
        
    }
    
};